flixel/system/FlxSound.hx

```diff
@@ -144,6 +144,10 @@ class FlxSound extends FlxBasic
 	 * Helper var to prevent the sound from playing after focus was regained when it was already paused.
 	 */
 	private var _alreadyPaused:Bool = false;
+	/**
+	 * In case of looping, from where to restart the sound when it loops back
+	 */
+	private var loopPoint(default, set):Float;
```

Comment:
This should be `public` (otherwise what's the point of making it a property with a setter?) and declared after `looping`.